<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalMappingDialog</class>
 <widget class="QDialog" name="SignalMappingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <layout class="QVBoxLayout" name="componentNameLayout">
       <item>
        <widget class="QLabel" name="componentNameLabel">
         <property name="text">
          <string>Component Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="componentNameList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="signalNameLayout">
       <item>
        <widget class="QLabel" name="signalNameLabel">
         <property name="text">
          <string>Signal Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="signalNameList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QPushButton" name="removeMappingButton">
       <property name="text">
        <string>Remove Mapping</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
